src/SortableContainer/index.js

```diff
@@ -241,7 +242,7 @@ export default function sortableContainer(WrappedComponent, config = {withRef: f
       }
     };
 
-    handlePress = event => {
+    handlePress = async event => {
```

Comment:
I bring this up only because I don't see `async/await` else where in the code base – is this within support range for the library?